in/unit/push-to-cloud.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=All-of-Us Arrays Push-to-Cloud Daemon
+After=network.target
+StartLimitIntervalSec=0
+
+[Service]
+EnvironmentFile=-/etc/sysconfig/push-to-cloud
+ExecStart=java -cp %PTC_BIN clojure.main -m ptc.start
+Restart=Always
+RestartSec=1
+SuccessExitStatus=143
+User=${USER}
```

Comment:
I'm not sure this is required for a user-level daemon

Reply:
It is not required for user level daemon.  It is required for a system level service running as a non-root user.